setup.cfg

```diff
@@ -19,3 +19,14 @@ max-line-length = 90
 #  W503 requires that binary operators be at the end, not start, of lines. Erik doesn't like it.
 #  E203 is contrary to PEP8.
 ignore = W503,E203
+
+[isort]
+line_length = 89
+not_skip = __init__.py
+sections=FUTURE,STDLIB,COMPAT,THIRDPARTY,TWISTED,FIRSTPARTY,LOCALFOLDER
+default_section=THIRDPARTY
+known_first_party = synapse
+known_compat = mock,six
+known_twisted=twisted,OpenSSL
```

Comment:
I'm not entirely convinced about putting mock, six, twisted and OpenSSL in separate groups. What's the thinking there?

Reply:
The reasoning for six being separate is that when we go Py3 only, that compat section gets removed (as six isn't needed and mock is in the stdlib in py3).

Twisted being separate is basically aesthetics for me, I added it after a test run to see how it would look. Since we import so much from it, it just adds a bit of whitespace between all those imports and the other third party ones, otherwise the list can get pretty dense.